src/components/ContentExplorer/ContentExplorer.js

```diff
@@ -1418,4 +1420,4 @@ class ContentExplorer extends Component<Props, State> {
 }
 
 export { ContentExplorer as ContentExplorerComponent };
-export default makeResponsive(ContentExplorer);
+export default withErrorBoundary(ERROR_TYPE_CONTENT_EXPLORER)(makeResponsive(ContentExplorer));
```

Comment:
I thought we were only modifying the EUA used elements?

Reply:
I added this because all of the preview test files actually use `ContentExplorer`